src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -299,6 +301,14 @@ subroutine atm_timestep(domain, dt, nowTime, itimestep, exchange_halo_group)
       call mpas_pool_get_array(state, 'xtime', xtime, 2)
       xtime = xtime_new
 
+      ! Get CF-compliant time at current timestep
+      call mpas_pool_get_array(state, 'Time', Time, 1)
+      Time_new = Time + dt
+
+      ! Write CF-compliant time for advanced timestep
+      call mpas_pool_get_array(state, 'Time', Time, 2)
+      Time = Time_new
```

Comment:
An alternative to consider for this new code might be the following:
```
      ! Update CF-compliant time at end of timestep
      call mpas_pool_get_array(state, 'Time', Time, 1)
      call mpas_pool_get_array(state, 'Time', Time_new, 2)
      Time_new = Time + dt
```
There's one less line of code, and it seems clearer to me. But I recognize that this is purely subjective, so I wouldn't object to the code as it is in the PR. Just a different way to consider. (Note that the variant code here requires `Time_new` to be declared with the `pointer` attribute.)

Reply:
I think I'm going to leave it as is for now, but this is certainly a more concise way to do this. I tried to follow the style of the xtime update in the lines above, which maybe helps with clarity if you're transitioning from using the xtime variable to the Time variable?

Reply:
In the case of the `xtime` update, the code is necessarily more complicated because we're updating a character string from a `real` and an `MPAS_Time_type`; on the other hand, in the case of `Time`, we're simply adding two `real` variables together. In my opinion, it's unnecessarily confusing to have the `Time` variable point to two different quantities only a few lines apart and to assign a value to `Time_new` only to then assign `Time_new` to `Time` (but not the same `Time` as a few lines earlier).